cli/tauri.js/test/jest/fixtures/app-test-setup.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path')
+const process = require('process')
+
+const appDir = path.resolve(__dirname, '../fixtures/app')
+const tauriDir = path.join(appDir, 'src-tauri')
+
+module.exports.appDir = appDir
+module.exports.distDir = path.join(appDir, 'dist')
+
+import * as appPaths from '../../../src/helpers/app-paths'
+
+module.exports.initJest = () => {
+  jest.mock('helpers/non-webpack-require', () => {
+    return path => {
+      const value = require('fs').readFileSync(path).toString()
+      if (path.endsWith('.json')) {
+        return JSON.parse(value)
+      }
+      return value
+    }
+  })
+  appPaths.appDir = appDir
+  appPaths.tauriDir = tauriDir
+  jest.spyOn(appPaths.resolve, 'app').mockImplementation(dir => path.resolve(appDir, dir))
+  jest.spyOn(appPaths.resolve, 'tauri').mockImplementation(dir => path.resolve(tauriDir, dir))
+}
+
+module.exports.startServer = (getAppPid, onReply) => {
+  const express = require('express')
+  const cors = require('cors')
+  const app = express()
```

Comment:
Express seems really overkill for this, why not use the [http.createServer](https://nodejs.org/api/http.html#http_http_createserver_options_requestlistener) included with node?